### ISSUE TITLE
Physically remove the hostile Chinese user base by removing Windows XP support

### ISSUE THREAD

**Issue by OrionsCore** [notepad-plus-plus/notepad-plus-plus#6484]
The attitude, hostilities, and threatening behaviour represents the anti-protestor gang.

I suggest removing support for Windows XP.

**Comment by michaellee8** [comment]
The intention seems good but IDK how is this related with XP.

**Comment by FANBOCHENG** [comment]
居然还觉得中国在用XP...这些傻逼

**Comment by dtwen-mk** [comment]
N M S L

**Comment by NainbowCat** [comment]
thats not enought, gfw should be banned in china so we can have a clean sight

**Comment by TechandMusic462** [comment]
> The intention seems good but IDK how is this related with XP.

It is known that the Chinese have pirated Windows XP enmasse over the last decade, and I guess the idea is that by removing support, a lot of Chinese lose access.

I'd just suggest putting in this, just so it gets banned in China:
动态网自由门 天安門 天安门 法輪功 李洪志 Free Tibet 六四天安門事件 The Tiananmen Square protests of 1989 天安門大屠殺 The Tiananmen Square Massacre 反右派鬥爭 The Anti-Rightist Struggle 大躍進政策 The Great Leap Forward 文化大革命 The Great Proletarian Cultural Revolution 人權 Human Rights 民運 Democratization 自由 Freedom 獨立 Independence 多黨制 Multi-party system 台灣 臺灣 Taiwan Formosa 中華民國 Republic of China 西藏 土伯特 唐古特 Tibet 達賴喇嘛 Dalai Lama 法輪功 Falun Dafa 新疆維吾爾自治區 The Xinjiang Uyghur Autonomous Region 諾貝爾和平獎 Nobel Peace Prize 劉暁波 Liu Xiaobo 民主 言論 思想 反共 反革命 抗議 運動 騷亂 暴亂 騷擾 擾亂 抗暴 平反 維權 示威游行 李洪志 法輪大法 大法弟子 強制斷種 強制堕胎 民族淨化 人體實驗 肅清 胡耀邦 趙紫陽 魏京生 王丹 還政於民 和平演變 激流中國 北京之春 大紀元時報 九評論共産黨 獨裁 專制 壓制 統一 監視 鎮壓 迫害 侵略 掠奪 破壞 拷問 屠殺 活摘器官 誘拐 買賣人口 遊進 走私 毒品 賣淫 春畫 賭博 六合彩 天安門 天安门 法輪功 李洪志 Winnie the Pooh 劉曉波动态网自由门

**Comment by OrionsCore** [comment]
@michaellee8 Just articles some time ago giving stats of XP use in China, @techandmusic462's idea is superior.

**Comment by OrionsCore** [comment]
@FANBOCHENG You're more than welcome to take a screenshot to "prove" that you, a single person, use something other than Windows XP.


Throwing this in for good measure:

动态网自由门 天安門 天安门 法輪功 李洪志 Free Tibet 六四天安門事件 The Tiananmen Square protests of 1989 天安門大屠殺 The Tiananmen Square Massacre 反右派鬥爭 The Anti-Rightist Struggle 大躍進政策 The Great Leap Forward 文化大革命 The Great Proletarian Cultural Revolution 人權 Human Rights 民運 Democratization 自由 Freedom 獨立 Independence 多黨制 Multi-party system 台灣 臺灣 Taiwan Formosa 中華民國 Republic of China 西藏 土伯特 唐古特 Tibet 達賴喇嘛 Dalai Lama 法輪功 Falun Dafa 新疆維吾爾自治區 The Xinjiang Uyghur Autonomous Region 諾貝爾和平獎 Nobel Peace Prize 劉暁波 Liu Xiaobo 民主 言論 思想 反共 反革命 抗議 運動 騷亂 暴亂 騷擾 擾亂 抗暴 平反 維權 示威游行 李洪志 法輪大法 大法弟子 強制斷種 強制堕胎 民族淨化 人體實驗 肅清 胡耀邦 趙紫陽 魏京生 王丹 還政於民 和平演變 激流中國 北京之春 大紀元時報 九評論共産黨 獨裁 專制 壓制 統一 監視 鎮壓 迫害 侵略 掠奪 破壞 拷問 屠殺 活摘器官 誘拐 買賣人口 遊進 走私 毒品 賣淫 春畫 賭博 六合彩 天安門 天安门 法輪功 李洪志 Winnie the Pooh 劉曉波动态网自由门

**Comment by dongyingt** [comment]
那，这根xp有什么关系？弱智了？好是觉着自己用的系统版本高？

**Comment by FANBOCHENG** [comment]
你先了解一下中国好么，别无脑的评论。带点脑子。我还需要证明我用不用XP？

**Comment by OrionsCore** [comment]
@dongyingt @FANBOCHENG You are claiming your shithole does not use Windows XP. You should back that up with proof, that's all. Stop being hostile shitheads.

If your shithole does not use Windows XP commonly, you're welcome to prove that, until then, your shithole still uses a lot of Windows XP machines.

**Comment by FANBOCHENG** [comment]
那你证明一下你发的那些新闻是真的好么？一样的道理有什么争的？我说你无脑是叫你别跟风懂了吗？ 是对你爱的鞭策。

**Comment by FANBOCHENG** [comment]
是你自己在挑事

**Comment by OrionsCore** [comment]
Speak English or fuck off. 

Can America please nuke China already? I fully advocate for the destruction of you and all other pro-China "people".

Either disconnect yourself, accept being disconnected, or eat shit.






> 是你自己在挑事



> 那你证明一下你发的那些新闻是真的好么？一样的道理有什么争的？我说你无脑是叫你别跟风懂了吗？ 是对你爱的鞭策。

**Comment by OrionsCore** [comment]
> 是你自己在挑事

![c7c](https://user-images.githubusercontent.com/56843849/67845776-f8d3b180-faf7-11e9-8a59-fd5001610af1.gif)

**Comment by FANBOCHENG** [comment]
啊哈哈哈哈 真是没脑子 请你说中文好吗？

**Comment by OrionsCore** [comment]
> 啊哈哈哈哈 真是没脑子 请你说中文好吗？

You are funny. I hope the radiation kills quickly.

**Comment by Tyil** [comment]
> The attitude, hostilities, and threatening behaviour represents the anti-protestor gang.

This was you, at the beginning.

> Can America please nuke China already? I fully advocate for the destruction of you and all other pro-China "people".

> You are funny. I hope the radiation kills quickly.

This is you, a couple moments later. You don't seem to be any better than "the anti-protestor gang".

Also, do you have any proof that removing Windows XP support will only affect "evil" Chinese users? I'd be surprised if those are the only people using Windows XP.

**Comment by FANBOCHENG** [comment]
我劝你们了解一下中国，完全不了解就在这里跟风真的像个小孩子。这里争论的其他中国人一样并没有说你们国家这样那样的谣言。不讲道理在这BB有什么意思？

**Comment by FANBOCHENG** [comment]
中国是非常安稳的国家，没有你们说的那么乱。别傻了被自己国家洗脑还在说别人不好？有个核弹好像了不起的样子。核弹都能搬出来真是搞笑。

**Comment by FANBOCHENG** [comment]
不喜欢就核平，不了解就核平，这就是你追求的民主自由？

**Comment by OrionsCore** [comment]
> > The attitude, hostilities, and threatening behaviour represents the anti-protestor gang.
> 
> This was you, at the beginning.
> 
> > Can America please nuke China already? I fully advocate for the destruction of you and all other pro-China "people".
> 
> > You are funny. I hope the radiation kills quickly.
> 
> This is you, a couple moments later. You don't seem to be any better than "the anti-protestor gang".
> 
> Also, do you have any proof that removing Windows XP support will only affect "evil" Chinese users? I'd be surprised if those are the only people using Windows XP.

These people do not deserve compassion or understanding, and they're still proving it now.

> Also, do you have any proof that removing Windows XP support will only affect "evil" Chinese users? I'd be surprised if those are the only people using Windows XP.

Collateral damage. @TechandMusic462's idea of distributing articles/text illegal in China with each copy of Notepad++ is superior to my idea anyway.

**Comment by OrionsCore** [comment]
> 不喜欢就核平，不了解就核平，这就是你追求的民主自由？

Are you incapable of posting a single comment instead of 3 or more?

**Comment by stevefan1999-personal** [comment]
lets stop arguing and watch some loli hentai

**Comment by OrionsCore** [comment]
> 不喜欢就核平，不了解就核平，这就是你追求的民主自由？
>[Do not like the nuclear level, you do not understand the nuclear level, this is your pursuit of democracy and freedom?]

If the wreckless, authoritarian government and people of China, with its organ harvesting, shitty manners, death threats, gutter oil food, brainwashed population, the inability to think critically of anything their government does or endorses, the social credit score system, and the fact none of you care about anything except posting and spamming for hours here, because again, you cannot think critically EVER of yourself or the government you love.

If the above gets destroyed, I am beyond comfortable with my country (no, not the US) being nuked as part of a retaliation because the government and the people of China (you included) cannot accept responsibility for being shitty people or even be try to be decent people, then fine.

Yes, I am genuinely more than happy to be vaporized/irradiated if China ceases to exist.

**Comment by OrionsCore** [comment]
Democracy and freedom is irrelevant if there's a chance for China's destruction.

**Comment by rulssi** [comment]
Wow, watching this Chinese meltdown is hilarious. This reaction really just proves that this was the right thing to do. Chinese people need to wake up, its strange how people from far away know more about their government and country than they do. Why their access here hasn't been blocked yet by their government surprises me. Nothing wrong with China as a country but the government and people from there really seem to suck. Write some more angry chinese noodles for me please! I'm gonna take a bath in your tears, there are way larger problems in the world than a fucking software developer saying something you don't like you stupid fucks. If you are here ranting and crying you should really fucking take a look in the mirror and realize what a useless shit you are. Go do something useful with your time instead of wasting it here.

**Comment by FANBOCHENG** [comment]
> Wow, watching this Chinese meltdown is hilarious. This reaction really just proves that this was the right thing to do. Chinese people need to wake up, its strange how people from far away know more about their government and country than they do. Why their access here hasn't been blocked yet by their government surprises me. Nothing wrong with China as a country but the government and people from there really seem to suck. Write some more angry chinese noodles for me please! I'm gonna take a bath in your tears, there are way larger problems in the world than a fucking software developer saying something you don't like you stupid fucks. If you are here ranting and crying you should really fucking take a look in the mirror and realize what a useless shit you are. Go do something useful with your time instead of wasting it here.

你们自己挑事 还不讲道理？你回复我不是在浪费时间？你在这发表意见？
你怎么不把你自己考虑进去就说别人浪费时间？脑子不会转？
你们在这挑事我们在这维护本身就是很正常的事情。
你看着你自己国家被造谣难道不管？

**Comment by OrionsCore** [comment]
> > Wow, watching this Chinese meltdown is hilarious. This reaction really just proves that this was the right thing to do. Chinese people need to wake up, its strange how people from far away know more about their government and country than they do. Why their access here hasn't been blocked yet by their government surprises me. Nothing wrong with China as a country but the government and people from there really seem to suck. Write some more angry chinese noodles for me please! I'm gonna take a bath in your tears, there are way larger problems in the world than a fucking software developer saying something you don't like you stupid fucks. If you are here ranting and crying you should really fucking take a look in the mirror and realize what a useless shit you are. Go do something useful with your time instead of wasting it here.
> 
> 你们自己挑事 还不讲道理？你回复我不是在浪费时间？你在这发表意见？
> 你怎么不把你自己考虑进去就说别人浪费时间？脑子不会转？
> 你们在这挑事我们在这维护本身就是很正常的事情。
> 你看着你自己国家被造谣难道不管？



Whatever buddy

Difference between you and I, I can be critical of my government, you cannot, just accept that and stop crying to people with irrelevant, vague, meaningless "arguments" that you managed to string together.

Cope more.

**Comment by FANBOCHENG** [comment]
> > 不喜欢就核平，不了解就核平，这就是你追求的民主自由？
> > [Do not like the nuclear level, you do not understand the nuclear level, this is your pursuit of democracy and freedom?]
> 
> If the wreckless, authoritarian government and people of China, with its organ harvesting, shitty manners, death threats, gutter oil food, brainwashed population, the inability to think critically of anything their government does or endorses, the social credit score system, and the fact none of you care about anything except posting and spamming for hours here, because again, you cannot think critically EVER of yourself or the government you love.
> 
> If the above gets destroyed, I am beyond comfortable with my country (no, not the US) being nuked as part of a retaliation because the government and the people of China (you included) cannot accept responsibility for being shitty people or even be try to be decent people, then fine.
> 
> Yes, I am genuinely more than happy to be vaporized/irradiated if China ceases to exist.

美国经常发生枪击案怎么不说？ 能不能不要再说那些有的没的 真的是没有意思
每个国家本身就存在很多问题。
你嘴里说的给我的感觉美国是天堂一样？
人从来不死，不发生恐袭，不示威，每个人都有钱？ 你敢说你们国家是这样的吗？

**Comment by rulssi** [comment]
> 
> 美国经常发生枪击案怎么不说？ 能不能不要再说那些有的没的 真的是没有意思
> 每个国家本身就存在很多问题。
> 你嘴里说的给我的感觉美国是天堂一样？
> 人从来不死，不发生恐袭，不示威，每个人都有钱？ 你敢说你们国家是这样的吗？

Everybody knows that US is a shithole so there's no need to say that. If I had to choose 3 countries to be erased I would say US, China and North Korea have to go. Yeah every country has problems but they don't spill them on other countries, well US does sometimes. But what your government is doing in Hongkong is so messed up that you might be in war within the the next few years. If you are defending your country you are delusional. I'm from Europe and I can say that my government is corrupted as fuck and there's many people that should be changed instantly. However its not as bad as other countries so I don't mind living in a free country with not much restrictions, however I would not defend my country if we were at war and I don't support or defend my goverment at the moment.

**Comment by OrionsCore** [comment]
> > > 不喜欢就核平，不了解就核平，这就是你追求的民主自由？
> > > [Do not like the nuclear level, you do not understand the nuclear level, this is your pursuit of democracy and freedom?]
> > 
> > 
> > If the wreckless, authoritarian government and people of China, with its organ harvesting, shitty manners, death threats, gutter oil food, brainwashed population, the inability to think critically of anything their government does or endorses, the social credit score system, and the fact none of you care about anything except posting and spamming for hours here, because again, you cannot think critically EVER of yourself or the government you love.
> > If the above gets destroyed, I am beyond comfortable with my country (no, not the US) being nuked as part of a retaliation because the government and the people of China (you included) cannot accept responsibility for being shitty people or even be try to be decent people, then fine.
> > Yes, I am genuinely more than happy to be vaporized/irradiated if China ceases to exist.
> 
> 美国经常发生枪击案怎么不说？ 能不能不要再说那些有的没的 真的是没有意思
> 每个国家本身就存在很多问题。
> 你嘴里说的给我的感觉美国是天堂一样？
> 人从来不死，不发生恐袭，不示威，每个人都有钱？ 你敢说你们国家是这样的吗？
>You say your country is that right?

No, never said that. I never said my country (as I said, I'm not from the US) is perfect.

Just because other countries are shitholes (most are) doesn't mean your country isn't. YOU are painting your country as perfect. It is not. It is a shithole, as are many others. Just yours is a special shithole because you will be murdered or enslaved for saying saying anything critical of your own government. Only a few others are like that (several African nations probably), but China is more technically advanced (because stealing IP :DD) and can do far worse shit to you if you defy the people you defend so much.

I'm not American, stop flailing and calm the fuck down. You're making a fool of yourself, using Whataboutism to defend your shithole country.

Read https://en.wikipedia.org/wiki/Whataboutism or https://zh.wikipedia.org/wiki/%E2%80%9C%E9%82%A3%E4%BD%A0%E4%BB%AC%E8%BF%98%E2%80%9D%E8%AE%BA 

I mean, if you can access Wikipedia of course.

I rarely defend my country, I only do so when shitheads like you think you have anything over us.

**Comment by FANBOCHENG** [comment]
我还是劝你来中国玩一下了解一下，然后就是不要想着删除谁。也没什么好说的中国人没那么好斗。

**Comment by OrionsCore** [comment]
Okay that Whataboutism article is blocked in China lmao.

![image](https://user-images.githubusercontent.com/56843849/67855836-22e29f00-fb0b-11e9-9da6-b7cb18742a5a.png)

**Comment by OrionsCore** [comment]
> 我还是劝你来中国玩一下了解一下，然后就是不要想着删除谁。也没什么好说的中国人没那么好斗。

>I have nothing to say Chinese people are not so aggressive.

No. You are wrong. This entire debacle and several other interactions I've had have proven that your government and your people are extremely aggressive. You cannot argue, articulate, or be nice at all in any situation that involves being critical of something.

You, and 3-4 other Chinese people I've interacted with in this repo, have ruined any good impression of Chinese people I had left (sending death threats for mentioning historical events tends to do that). 

Congratulations, I doubt I'm alone. Go eat more gutteroil enriched food or whatever the fuck you do when not shilling.

**Comment by FANBOCHENG** [comment]
You're so paranoid. I'm off work. Bye.

**Comment by OrionsCore** [comment]
> You're so paranoid. I'm off work. Bye.

Nope, you just can't cope so you flail and threaten death. You are a scumbag and I hope you are obliterated by your government, hopefully seconds after realizing how  pathetic you are.

>Bye.

Actually, that isn't so easy.

Talk soon.

**Comment by FANBOCHENG** [comment]
...
你固执到一定程度了

**Comment by yur3i** [comment]
> > You're so paranoid. I'm off work. Bye.
> 
> Nope, you just can't cope so you flail and threaten death. You are a scumbag and I hope you are obliterated by your government, hopefully seconds after realizing how pathetic you are.
> 
> > Bye.
> 
> Actually, that isn't so easy.
> 
> Nothing I said is paranoia, you are, again, using insults that don't fit anywhere.
> 
> Talk soon

Hoes mad

**Comment by OrionsCore** [comment]
stfu yuri 
let me masturbate to Chinese arguments in peace

**Comment by OrionsCore** [comment]
> ...
> 你固执到一定程度了

Yeah, you are too, you've been at this for 3-4 hours now too, hypocrite.

**Comment by Willt125** [comment]
Why the hell are we having this argument on a repo, go to Twitter or something. Leave the devs in peace.
